hnn_core/dipole.py

```diff
@@ -64,8 +64,8 @@ def simulate_dipole(net, tstop, dt=0.025, n_trials=None, record_vsec=False,
                       'create a predefined network from published models.',
                       UserWarning)
     # ADD DRIVE WARNINGS HERE
-    if not net.external_drives:
-        warnings.warn('No external drives loaded', UserWarning)
+    if not net.external_drives and not net.external_biases:
+        warnings.warn('No external drives or biases loaded', UserWarning)
```

Comment:
I didn't think this warning should be showing up in the `test_tonic_biases()` test. Any objections?

Reply:
agreed